earthfile2llb/converter.go

```diff
@@ -18,6 +18,9 @@ import (
 	"strings"
 	"time"
 
+	"github.com/alessio/shellescape"
```

Comment:
can you move these to the next section down.

Reply:
done